extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -28,7 +28,8 @@ public static function onAfterInitialize( $title, $article, $output, $user, WebR
 		global $wgAdDriverForceDirectGptAd, $wgAdDriverForceLiftiumAd,
 			   $wgLiftiumOnLoad, $wgNoExternals, $wgEnableKruxTargeting,
 			   $wgAdEngineDisableLateQueue, $wgLoadAdsInHead, $wgLoadLateAdsAfterPageLoad,
-			   $wgEnableKruxOnMobile, $wgAdDriverForceTurtleAd, $wgAdDriverUseSevenOneMedia, $wgUsePostScribe;
+			   $wgEnableKruxOnMobile, $wgAdDriverForceSzymonAd, $wgAdDriverForceTurtleAd, $wgAdDriverUseSevenOneMedia,
```

Comment:
I don't know how and when it happen but we became fans of alphabetical order. You did a good job here with placing `$wgAdDriverForceSzymonAd` before `$wgAdDriverForceTurtleAd` but since we're here we can sort the whole "thing". So, let's put `$wgAdEngine*` stuff first, then `$wgAdDriver*` and so on... :japanese_ogre: